### DISCUSSION TITLE
Monthly Data Notifications to Slack

### DISCUSSION THREAD

**Discussion by DOGUSENSI** [planetscale/discussion#561] [Q&A] [answered]
Is there a way to send monthly read/write counts and storage space to Slack?

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @DOGUSENSI,

I have a couple ideas for you! 

- First, for storage, we have [a weekly email report](https://planetscale.com/blog/introducing-database-reports) that shows slow queries and DB usage. That could be auto forwarded to Slack.
- Another option would be the [Datadog integration](https://planetscale.com/docs/integrations/datadog). With it, you'd get your PlanetScale read/write/storage data available in Datadog. You should then be able to get that data into Slack via the Datadog + Slack integration.

With datadog, it could also be useful for setting up custom alerts around usage. Which may be better than a monthly notification.

**Reply by DOGUSENSI** [reply]
Hello, @mscoutermarsh !

Thanks for the interesting and practical ideas. I was only thinking of getting the information through the webAPI provided by planetscale. Both ideas are appealing, but I will consider using datadog.
